help/status-and-availability.md

```diff
@@ -32,6 +32,16 @@ You can set a status emoji, status message, or both.
 
 1. Click **Save**.
 
+{tab|via-personal-settings}
+
+1. Click your profile picture on the top right
```

Comment:
All instructions should end in a period as imperative sentences.

Reply:
I've added a period. I will refrain from making these kinds of mistakes in the future